.prettierignore

```diff
@@ -0,0 +1,2 @@
+/dist
+/src/util/apis/generated
```

Comment:
can we have a newline here? (if prettier accepts)

Reply:
Oh yeah of course. Prettier actually enforces that, but doesn't run on this file type.

Reply:
Nope it does not enforce that... I must be thinking of some other tool. But I'll add it, no problem.

Reply:
Done!